CWE/inventions/navy_inventions.txt

```diff
@@ -785,39 +785,39 @@ OR = { has_country_modifier = spcno has_country_modifier = military_tech_coopera
 }
 
 #destroyers, 1946
-modern_destroyers_invention = {
-	limit = { corvettes = 1 }
-	chance = {
-		base = 0
-		modifier = {
-			factor = 4
-			has_country_modifier = wonder_weapons
-		}
-		modifier = {
-			factor = 2
-			military_spending = 0.50
-		}
-		modifier = {
-			factor = 2
-			military_spending = 0.75
-		}
-		modifier = {
-			factor = -2
-			is_greater_power = no 
-			is_secondary_power = no
-		}
-		modifier = {
-			factor = 2
-OR = { has_country_modifier = spcno has_country_modifier = military_tech_cooperation }
-		}
-	}
-	effect = {
-	
-	activate_unit = destroyer
-	
-	}
-
-}
+#modern_destroyers_invention = {
```

Comment:
This invention cannot be commented out because destroyers are activated solely by this invention.

Reply:
The destroyer is marked active = yes so it should be available from the start.

Reply:
Strange, I must have forgotten to mark it as active = no .

Reply:
A lot of navies start with destroyers in 1946; should I replace them with something else, like the patrol boat?

Reply:
I think you can just add the tech that activates the invention to their country history file instead as the AI loves to immediately disband all patrol boats once the game starts.

Reply:
The destroyer invention isn't commented out anymore. I'll make any further changes to navy techs in a separate PR.